### PR TITLE
Add Components to Redux (PR)

### PR DESCRIPTION
**Pull request by teke85**
## In this project milestone, the following requirements were implemented:

- First of all - make sure that you keep state immutable in reducers!
- Secondly, change the default state in your books reducer from an empty array to an array with a few books. One of the goals is to display them in your React components.
- Import Redux Provider and your store in the main component of your React app.
- Connect your app to the Redux store.
- Make sure that list of books displays books from Redux store.
- Use useSelector to consume the state.
- In your React component responsible for adding new books, add functionality for "Add new" button click:
- Import Redux useDispatch.
- Set the data inputs in the local React state (set title and author - NOTE: categories and comments are NOT part of this step).
- Takes the inputs from the state and generate a unique id and add them to an object.
- Once your new book object is ready to be submitted to Redux store, dispatch a corresponding action (import Action Creator that you created before and use it here).
- In your React component responsible for removing books, add a functionality for "Remove" button click:
- Import Redux useDispatch.
- Dispatch a corresponding action (import Action Creator that you created before and use it here).
- In your React component responsible for categories, add a functionality for "Check status" button click:
- Import Redux useDispatch.
- Dispatch a corresponding action (import Action Creator that you created before and use it here).

### CODE REVIEW

File: src/Components/Categories.js [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
+  const dispatch = useDispatch();
+  return (
+    <div>
+      <button type="button" onClick={() => dispatch(checkStatus())}>
+        Check Status
+      </button>
+      <h2>{categories}</h2>
+    </div>
+  );
+};
```
Comment (GzimAsani) [inline src/Components/Categories.js:16]: - [ ] Kindly check this error on the categories page. Update and remove the error 
image

Comment (GzimAsani) [reply src/Components/Categories.js:16]: - [ ] The logo is broken on your readme file, kindly update it 
image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello @teke85                    :wave:   
 Good job so far,  done There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with the redux :+1: 

 ## Required Changes 
- [ ] Check the comments below :arrow_double_down: 


 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Components/Categories.js (hunk 1 above)
Comment (teke85) [reply src/Components/Categories.js:16]: > * [ ]  Kindly check this error on the categories page. Update and remove the error
>   image

Hi @GzimAsani Thank you very much for your feedback. I have fixed the errors on the categories page

Comment (teke85) [reply src/Components/Categories.js:16]: > * [ ]  The logo is broken on your readme file, kindly update it
>   image

Hey @GzimAsani Thank you for the observation. I have fixed the logo

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @teke85 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏
- [ ] it would be better to make sure that the `MIT` link is working and it should take you to the License of the project in order for your project will be professional
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**